Library/Homebrew/manpages/brew.1.md.erb

```diff
@@ -61,12 +61,6 @@ With `--verbose` or `-v`, many commands print extra debugging information. Note
 
   Homebrew/homebrew-bundle <https://github.com/Homebrew/homebrew-bundle>
 
-  * `cask` [`--version` | `audit` | `cat` | `cleanup` | `create` | `doctor` | `edit` | `fetch` | `home` | `info`]:
```

Comment:
Please add this back.

Reply:
Added it to the `COMMANDS` section, since it's not an external command anymore.

Reply:
It still acts very much as an external command even if it's inside Homebrew/brew and, like the other external commands, it immediately taps a tap to be functionality. I'd rather it was left as-is until the `brew-cask.1` manpage is entirely moved into `brew.1`.

Reply:
> It's not like external commands, for external commands, you first have to tap a tap to use the command, for brew cask you don't, it just happens to install a tap as a dependency on first use

For the user this is functionally identical to `bundle` and `services` both of which happen to install a tap as a dependency on first use. 

If we get to a point where we have https://caskroom.github.io merged into https://brew.sh, `brew cask --help` resembling `brew install --help` and other similar style matching and `brew-cask.1` goes away I'll be 👍 on changing this. Until then: please put it back how it was until we're further along, thanks.

Reply:
> For the user this is functionally identical to `bundle` and `services` both of which happen to install a tap as a dependency on first use. 

I fail to see how the functionality of a command has anything to do with it being external. I can have an internal command printing “Hello, world!" or an external command printing "Hello, world!".

Reply:
Please put it back to where it was and we can revisit it later (as I've stated multiple times above). This change is unrelated to the rest of the PR, your GSoC project and is not a good use of either of our time to go around in circles about it.